smart-contracts/contracts/interfaces/IUnlock.sol

```diff
@@ -76,13 +76,13 @@ interface IUnlock {
     returns (uint discount, uint tokens);
 
   // Function to read the globalTokenURI field.
-  function getGlobalBaseTokenURI()
```

Comment:
Isn't this required by erc721 spec though?

Reply:
`tokenURI` is the expected function.  This is just an internal helper we use to support that call.